src/main/kotlin/graphics/scenery/proteins/RibbonDiagram.kt

```diff
@@ -522,25 +521,60 @@ class RibbonDiagram(val protein: Protein, private val displaySS: Boolean = false
         }
     }
 
+
     /**
-     * Counts the secondary structure length.
+     * We only differentiate between sheets, helices, and coils
      */
-    private fun getCount(guidePointList: List<GuidePoint>): Int {
-        var count = 0
-            guidePointList.forEachIndexed { index, guide ->
-                if (index < guidePointList.lastIndex) {
-                    val nextGuide = guidePointList[index + 1]
-                    //Secondary structures which are not sheets or helices are summarized
-                    if (guide.type == SecStrucType.helix4 && nextGuide.type == SecStrucType.helix4 ||
-                            guide.type.isBetaStrand && nextGuide.type.isBetaStrand ||
-                            (guide.type != SecStrucType.helix4 && nextGuide.type != SecStrucType.helix4
-                                    && !guide.type.isBetaStrand && !nextGuide.type.isBetaStrand)){
-                        count++
-                    } else {
-                        return count
+    enum class SimplifiedType { sheet, helix, coil }
```

Comment:
Members of enums should be capitalised (e.g. `Sheet`, `Helix`, etc.)

Reply:
Check. The enum became superfluous anyway, but thanks for the hint!